tests/test_workers/test_tasks/test_build.py

```diff
@@ -436,6 +438,8 @@ def test_handle_add_request(
     mock_cleanup,
     mock_ugrb,
     mock_dep_b,
+    mock_sir,
```

Comment:
Not sure if IIB has a requirement on this, but shouldn't unit tests be added for the two new functions?

Reply:
happy to add unit tests - will take a look at that today!